CliClient/locales/ko.po

```diff
@@ -37,16 +39,16 @@ msgid "No notebook has been specified."
 msgstr "특정된 노트북이 없습니다."
 
 msgid "Y"
-msgstr "Y"
+msgstr "예"
 
 msgid "n"
-msgstr "n"
+msgstr "아니오"
 
 msgid "N"
-msgstr "N"
+msgstr "아니오"
 
 msgid "y"
-msgstr "y"
+msgstr "예"
```

Comment:
These are not to be translated. AFAIK, they are used to answer questions. If some enters a Korean character nothing will happen, because the code checks for y, Y, n, N.

@laurent22 maybe we should add comments for translators. It is possible with poedit.